src/libraries/Microsoft.PowerFx.Connectors/Execution/OpenApiTextSerializer.cs

```diff
@@ -72,21 +76,26 @@ protected override void WriteDecimalValue(decimal decimalValue)
         }
 
         protected override void WritePropertyName(string name)
-        {            
+        {
         }
 
         protected override void WriteStringValue(string stringValue)
         {
             _writer.Append(stringValue);
-        }     
+        }
+
+        protected override async void WriteBlobValue(BlobValue blobValue)
+        {
+            _writer.Append(await blobValue.GetAsStringAsync(null, _cancellationToken).ConfigureAwait(false));
```

Comment:
>GetAsStringAsync [](http://example.com/codeflow?start=43&length=16)

What is the scenario where this code would be executed?

Reply:
Seems to be with `AzureBlobStorage.CreateFile("container", "bora4.txt", AsBlob("abc")).Size` type of expression. 
I'll add a test with a JPG and see if the result if correct